specification/gedcom-3-structures-3-meaning.md

```diff
@@ -907,28 +907,55 @@ As of version 7.0, only 2 media types are supported by this structure:
     Supporting more of HTML is encouraged.
     Unsupported tags should be ignored during display.
 
-:::note
-Applications are welcome to support more XML entities or HTML character references in their user interface.
-However, exporting must only use the core XML entities, translating any other entities into their corresponding Unicode characters.
-:::
+    <div class="note">
+    Applications are welcome to support more XML entities or HTML character references in their user interface.
+    However, exporting must only use the core XML entities, translating any other entities into their corresponding Unicode characters.
+    </div>
 
-:::note
-Applications are welcome to support additional HTML elements,
-but they should ensure that content is meaningful if those extra elements are ignored and only their content text is displayed.
-:::
+    <div class="note">
+    Applications are welcome to support additional HTML elements,
+    but they should ensure that content is meaningful if those extra elements are ignored and only their content text is displayed.
+    </div>
 
+    If needed, `text/html` can be converted to `text/plain` using the following steps:
 
-:::note
-Media types are also used by external files, as described under `FORM`. External file media types are not limited to `text/plain` and `text/html`.
-:::
+    1. Replace any sequence of 1 or more spaces, tabs, and line breaks with a single space
+    2. Case-insensitively replace each `<p`...`>`, `</p`...`>`, and `<br`...`>` with a line break
+    3. Remove all other `<`...`>` tags
+    4. Replace each `&lt;` with `<` and `&gt;` with `>`
+    5. Replace each `&amp;` with `&`
+
+- `text/markdown` employs a plain text format for creating structured documents, and is designed
+  to be fairly understandable by users even if only displayed as plain text.
+  Applications should support at least the following:
 
-If needed, `text/html` can be converted to `text/plain` using the following steps:
+    - `**`, `*`, `__`, and `~~` for bold, italic, underlined, and strike-through text.
```

Comment:
`~~` is not part of the originak Markdown definition nor of commonmark; while useful, it is an extension and perhaps does not belong in this spec